server/src/main.rs

```diff
@@ -24,12 +24,16 @@ async fn main() -> std::io::Result<()> {
         .build(manager)
         .expect("Failed to create pool.");
 
+    let server_port = std::env::var("SERVER_PORT").expect("SERVER_PORT must be set");
+    let mut server_address = "0.0.0.0:".to_string();
+    server_address.push_str(&server_port[..]);
```

Comment:
We could either specify the whole address in the `.env` or the listen address as well. This would make it easier to change it if one does not want to have the server listen on all interfaces.

Regarding the string building here, the [format!](https://doc.rust-lang.org/std/macro.format.html) macro is probably what you want here.

```rust
format!("0.0.0.0:{}", server_port)
```